templates/desktop/developers.html

```diff
@@ -501,6 +501,32 @@ <h3>Certified hardware</h3>
       </div>
     </div>
   </section>
+  <section class="p-strip--light">
+    <div class="row">
+      <div class="col-8">
+        <h2>What's new in Ubuntu {{ releases.lts.short_version }}</h2>
+      </div>
+    </div>
+    <div class="u-fixed-width">
+      <ul class="p-list is-split">
+        <li class="p-list__item is-ticked">Linux 5.19 kernel</li>
+        <li class="p-list__item is-ticked">GNOME 43 desktop environment
+          <ul>
+            <li>New quick settings interface</li>
+            <li>New spread effect for multitasking</li>
+            <li>Dynamic UI for the file manager</li>
+          </ul>
+        </li>
+        <li class="p-list__item is-ticked">Pipewire as the default audio system delivering improve bluetooth performance</li>
+        <li class="p-list__item is-ticked">Up-to-date toolchains, including Python 3.10, PHP 8.1, Ruby 3.1, Perl 5.34, Go 1.18, GCC 12.2, Rust 1.59</li>
+        <li class="p-list__item is-ticked">Core productivity apps LibreOffice 7.4, Thunderbird 102, and Firefox 104</li>
+        <li class="p-list__item is-ticked">Multi-threaded CPU squashfs decompression for faster Snap startup</li>
+      </ul>
+    </div>
+    <div class="u-fixed-width">
+      <p><a href="{{ releases.lts.release_notes_url }}">Read the full release notes&nbsp;&rsaquo;</a></p>
+    </div>
+  </section>
```

Comment:
Do you think we need to add a line between 22.10 section and 22.04 section? 
```suggestion
  </section>
   <div class="u-fixed-width">
      <hr class="p-separator" />
    </div>
```

Reply:
Added

Reply:
well, I'd rather not if they are going to take 2 days to review a line...